src/cargo/core/shell.rs

```diff
@@ -204,6 +225,16 @@ impl Shell {
         self.print(&status, Some(&message), Green, true)
     }
 
+    /// Shortcut to right-align and paint the query part of the output message as a green in stdout.
+    pub fn status_stdout_part_green(
```

Comment:
Given how specific this method is to `cargo search` I think it would be better to implement this in the `cargo search` area rather than in the general shell area.

Reply:
thank you, i have changed it.